EventsExpress/ClientApp/src/services/AuthenticationService.js

```diff
@@ -3,6 +3,9 @@ import EventsExpressService from './EventsExpressService';
 const baseService = new EventsExpressService();
 
 export default class AuthenticationService {
+    getCurrentToken = () => localStorage.getItem('token');
+    
+    setAuth = () => baseService.setResource('Authentication/login_token');
```

Comment:
I guess we could drop this behavior cause we have a refresh-token strategy instead.

Reply:
@iuriikhrystiuk said it would be nice to incapsulate this logic in some service